src/d3a/models/strategy/pv.py

```diff
@@ -106,7 +106,7 @@ def produced_energy_forecast_kWh(self):
         # This forecast ist based on the real PV system data provided by enphase
         # They can be found in the tools folder
         # A fit of a gaussian function to those data results in a formula Energy(time)
-        for slot_time in generate_market_slot_list(self.area):
+        for slot_time in generate_market_slot_list(area=self.area):
```

Comment:
Does preferring kwargs over args also help in reducing RAM requirement?

Reply:
No, this is just because I changed the implementation of `generate_market_slot_list`.